### ISSUE TITLE
Mail app: let table navigation commands be used to review message headers

### ISSUE THREAD

**Issue by josephsl** [josephsl/wintenApps#11]
Hi,

Similar to Outlook, NVDA should be able to use table nav commands in Mail app to review message headers. Thankfully, this is quite simple, as all the conditions to treat mail items as fake table row are there (children are columns). For now, a prototype will be included in WinTenApps, but it should become part of NVDA Core in 2017.2.

Thanks.

**Comment by josephsl** [comment]
Hi,

Implemented inside the add-on, no NVDA Core pull request planned.

**Comment by josephsl** [comment]
Hi,

 2022 update: with a long-term strategy from Microsoft to replace Mail and Calendar with One Outlook (Project Monarch), this feature might be removed in a future add-on release (as early as copper 4 milestone/2023), also to reduce maintenance burden.

Thanks.
